src/mockLink.test.ts

```diff
@@ -242,4 +237,46 @@ describe('class MockLink', () => {
       expect(observer.complete).toBeCalledTimes(1);
     });
   });
+
+  describe('constructor option "missingHandlerPolicy"', () => {
+    it('when "throw-error" throws when a handler is not defined for the query', () => {
+      mockLink = new MockLink({missingHandlerPolicy: 'throw-error'})
+
+      expect(() => mockLink.request(queryOneOperation))
+        .toThrowError(`Request handler not defined for query: ${print(queryOne)}`)
+    });
+
+    it('when "warn-and-return-error" logs a warning when a handler is not defined for the query', async () => {
+      mockLink = new MockLink({missingHandlerPolicy: 'warn-and-return-error'})
+
+      const observable = mockLink.request(queryOneOperation);
+      const observer = createMockObserver();
+
+      observable.subscribe(observer);
+
+      await new Promise(r => setTimeout(r, 0));
+
+      expect(observer.next).not.toBeCalled();
+      expect(observer.error).toBeCalled();
+      expect(observer.complete).not.toBeCalled();
+      expect(console.warn).toBeCalledTimes(1);
+      expect(console.warn).toBeCalledWith(`Request handler not defined for query: ${print(queryOne)}`);
+    });
+
+    it('when "ignore" returns an error when a handler is not defined for the query', async () => {
```

Comment:
Very minor - I think `ignore` should be `return-error` in the test description here?

Reply:
thanks for noticing! I've updated it to be `return-error`

Reply:
thanks for noticing! I've updated it to be return-error